src/main/resources/assets/botania/lang/fr_FR.lang

```diff
@@ -121,34 +140,35 @@ botania.rank5=&6SS
-botania.wings6=Wings: Lord of the Pit
-botania.wings7=Wings: Sylphide
+botania.wings1=Ailes : Flügel
+botania.wings2=Ailes : Ange uni-ailé
+botania.wings3=Ailes : Féerie glacée
+botania.wings4=Ailes : Seigneur de feu
+botania.wings5=Ailes : Princesse noire neige
+botania.wings6=Ailes : Lord of the Pit
+botania.wings7=Ailes : Sylphide
+botania.wings8=Ailes : Les seules
 
 # ARMOR SETS
-botania.armorset.manasteel.name=Mana Affinity
-botania.armorset.manasteel.desc=10% less Mana cost on Mana Tools
-botania.armorset.elementium.name=Great Fairy's Blessing
+botania.armorset.manasteel.name=Affinité au Mana 
+botania.armorset.manasteel.desc=Coût en Mana diminué de 10% sur les outils à Mana
+botania.armorset.elementium.name=Bénédiction de la reine fée
 botania.armorset.elementium.desc=Chance for pixies to apply potion effects
-botania.armorset.terrasteel.name=Will of the Ancients
-botania.armorset.terrasteel.desc0=20% less Mana cost on Mana Tools
-botania.armorset.terrasteel.desc1=Regen even if hunger isn't topped off
-botania.armorset.will0.desc=Critical Hits apply a strong weakness effect (Will of Ahrim)
-botania.armorset.will1.desc=Critical Hit damage increases with lower HP (Will of Dharok)
-botania.armorset.will2.desc=Critical Hits heal for a portion of damage dealt (Will of Guthan)
-botania.armorset.will3.desc=Critical Hits apply a strong slowness effect (Will of Torag)
-botania.armorset.will4.desc=Critical Hits pierce through armor (Will of Verac)
-botania.armorset.will5.desc=Critical Hits apply a wither effect (Will of Karil)
-botania.armorset.will0.shortDesc=Critical Hits apply a strong weakness effect
-botania.armorset.will1.shortDesc=Critical Hit damage increases with lower HP
-botania.armorset.will2.shortDesc=Critical Hits heal for a portion of damage dealt
-botania.armorset.will3.shortDesc=Critical Hits apply a strong slowness effect
-botania.armorset.will4.shortDesc=Critical Hits pierce through armor
-botania.armorset.will5.shortDesc=Critical Hits apply a wither effect
+botania.armorset.terrasteel.name=Savoir des Ancients
+botania.armorset.terrasteel.desc0=Coût en Mana diminué de 20% sur les outils à Mana
+botania.armorset.terrasteel.desc1=Régène même si la faim n'est pas remplie
+botania.armorset.will0.desc=Les coups critiques appliquent un effet de faiblesse fort (Volonté d'Ahrim)
+botania.armorset.will1.desc=Les coups critiques augmentent avec un HP bas (Volonté de Dharok)
+botania.armorset.will2.desc=Les coups critiques guérit pour une partie des dégâts infligés (Volonté de Guthan)
+botania.armorset.will3.desc=Les coups critiques appliquent un effet de lenteur fort (Volonté de Torag)
+botania.armorset.will4.desc=Les coups critiques transperce l'armure (Volonté de Verac)
+botania.armorset.will5.desc=Les coups critiques appliquent un effet de wither (Volonté de Karil)
+botania.armorset.will0.shortDesc=Les coups critiques appliquent un effet de faiblesse fort
+botania.armorset.will1.shortDesc=Les coups critiques augmentent avec un HP bas
```

Comment:
"Les coups critiques augmentent avec un HP bas"
Sachant que HP = Heal Point = point de vie
"Les coups critiques augmentent avec un point de vie bas", ça ne va pas.

Par contre, "Les coups critiques augmentent quand les points de vies sont bas" ça marche mieux.

This is the last time that I will use french in a PR.